components/src/atoms/button/Button.html

```diff
@@ -0,0 +1,7 @@
+<button
+  :class="{'button': defaultClass}"
+  @click="onClick"
+>
```

Comment:
- we don't need `@click` event here. 
- `button` class should be added without condition